src/process/basic_process.cpp

```diff
@@ -178,6 +180,10 @@ mp::ProcessState mp::BasicProcess::execute(const int timeout)
     mp::ProcessState exit_state;
     start();
 
+    mpl::log(mpl::Level::debug, "basic_process",
+             fmt::format("executing {} {}", qUtf8Printable(process_spec->program()),
```

Comment:
```suggestion
             fmt::format("executing '{} {}'", qUtf8Printable(process_spec->program()),
```

Reply:
I thought about that, but didn't use quotes in the log because in case the command has some sort of quotes it may be confusing.

Reply:
Anyway, having quotes separates the command from the rest, and the line can be parsed even with quotes. I accept the suggestion.